packages/cms/pages/settings/team-members/team-members.tsx

```diff
@@ -68,18 +77,162 @@ const UserRole = styled.span`
 `
 
 interface ProfileProps {}
+interface FlyOutProps {
+  setIsFlyoutVisible: (b: boolean) => void
+  selectedMember: TeamMemberProps
+  showChangeMemberRoleModal: () => void
+  getTeamMembers: () => void
+}
+const FlyOut: React.FC<FlyOutProps> = ({
+  setIsFlyoutVisible,
+  selectedMember,
+  showChangeMemberRoleModal,
+  getTeamMembers
+}) => {
+  const simpleFlyoutTitleId = useGeneratedHtmlId({
+    prefix: 'simpleFlyoutTitle'
+  })
+
+  const [memberDetails, setMemberDetails] = useState({
+    firstName: selectedMember.firstName,
+    lastName: selectedMember.lastName,
+    password: ''
+  })
+  const [confirmPassword, setConfirmPassword] = useState('')
```

Comment:
I think we should leave out confirm password, since this feature is mainly used by administrators resetting the password of another admin who is locked out of their account.